src/Take.Blip.Builder/Models/Input.cs

```diff
@@ -10,7 +11,8 @@ namespace Take.Blip.Builder.Models
     /// </summary>
     public class Input : IValidable
     {
-        private static readonly Regex VariableValidationRegex = new Regex("^([a-zA-Z0-9\\.]+)$", RegexOptions.Compiled);
+        private static IConfiguration _configuration = new ConventionsConfiguration();
```

Comment:
You must not initialize instances, it should be made by the service provider

Reply:
Since its a simple regex as said below, we will not validate the timeout